### PR TITLE
AF-2880 : Support Elytron in Business Central

### PR DESCRIPTION
**Pull request by Rikkola**
**Thank you for submitting this pull request**

**JIRA**: 

[AF-2880](https://issues.redhat.com/browse/AF-2880)

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1723
* https://github.com/kiegroup/droolsjbpm-integration/pull/2559
* https://github.com/kiegroup/kie-soup/pull/220
* https://github.com/kiegroup/appformer/pull/1182
* https://github.com/kiegroup/drools-wb/pull/1519
* https://github.com/kiegroup/jbpm-wb/pull/1511
* https://github.com/kiegroup/kie-wb-distributions/pull/1121

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by Rikkola** [comment]
Jenkins run fdb

**Comment by Rikkola** [comment]
Jenkins run fdb

**Review by mareknovotny — APPROVED** [review]

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by Rikkola** [comment]
jenkins do fdb

**Comment by Rikkola** [comment]
jenkins do fdb

**Comment by mbiarnes** [comment]
Jenkins run fdb

File: pom.xml [hunk 1]
```diff
@@ -430,7 +430,7 @@
     <!-- Version of the KIE Workbench application. Shown for example in About dialog. Will be usually overridden by productisation. -->
     <version.org.kie.workbench.app>${version.org.kie}</version.org.kie.workbench.app>
 
-    <version.thorntail>2.7.0.Final</version.thorntail>
+    <version.bootable>4.0.3.Final</version.bootable>
```
Comment (mareknovotny) [inline pom.xml:433]:
```suggestion
    <version.bootable>4.1.0.Final</version.bootable>
```

Comment (mareknovotny) [reply pom.xml:433]: @Rikkola you should update the maven plugin

Comment (mareknovotny) [reply pom.xml:433]: committing it for now

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by Rikkola** [comment]
jenkins do fdb

**Comment by Rikkola** [comment]
I added the bootable jar changes in. The two more or less need each other.

**Comment by Rikkola** [comment]
Jenkins run fdb

**Comment by mbiarnes** [comment]
@Rikkola previous fdb didn't work because of a already fixed failure in a script.

Jenkins run fdb

**Comment by mareknovotny** [comment]
jenkind do fdb

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by Rikkola** [comment]
Jenkins run fdb

**Review by tomasdavidorg — APPROVED** [review]
Approving. (It was decided that EAP issues will be resolved later.)

**Comment by Rikkola** [comment]
Jenkins run fdb

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
ok i can see all test failures in Karaf or as in PR CI of kie-wb-distribution https://github.com/kiegroup/kie-wb-distributions/pull/1121#issuecomment-939463049 (`org.kie.wb.test.rest.functional.ProjectIntegrationTest.testDeployProjectSnapshotTwice`)

**Comment by mbiarnes** [comment]
Jenkins run jdk11-fdb

**Pull request merged**